.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Hi there! Thank you for discovering and submitting an issue!
+
+Note: please use your Azure subscription if you need to share any information from your Azure subscription such as connection strings, service names (IoTHub, Provisioning), etc.
+-->
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Steps to Reproduce
+<!-- Provide a detailed set of steps to reproduce the bug. -->
+1.
+2.
+3.
+4.
+
+## Context (Environment)
+
+### Device (Host) Operating System
+<!-- Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core, etc. -->
+
+### Architecture
+<!-- armv7, arm32, x86 etc. -->
```

Comment:
@ronniesa  Should the SPI dev version be mentionned here in case of x86

Reply:
no need for this we can always ask if we think the problem could be related..... but it may confuse customers too